src/Makefile

```diff
@@ -0,0 +1,18 @@
+BINDIR = ${DESTDIR}/usr/bin
+CC = gcc
```

Comment:
You don't need to set CC, CC is a builtin variable.

Reply:
Sure, but the default value is 'cc' and I wanted to be verbose about using gcc. Not that it should have any effect on the result, so I can remove this if it's generally considered bad practise.

Reply:
Yeah I think so, it makes sense to stick to the upstreams defaults.